KsApi/models/DeletePaymentMethodEnvelopeTests.swift

```diff
@@ -19,6 +19,7 @@ class DeletePaymentMethodEnvelopeTests: XCTestCase {
     let data = jsonString.data(using: .utf8)
 
     do {
+      //swiftlint:disable force_unwrapping
```

Comment:
Are you getting a Swiftlint warning here? Is this by any chance a newer version of Swiftlint than what we're using (I think we hardcoded the build script to use `0.29.2`).

Reply:
I'm not, but that's because swiftlint is currently broken 😬 once its fixed, this line is needed